Web/src/components/AddExpenseDialog.jsx

```
@@ -0,0 +1,225 @@
+            : ['date', 'description', 'category', 'account', 'amount'];
+
+        const missingFields = requiredFields.filter(field => !formData[field]);
+        
+        if (missingFields.length > 0) {
+            alert(`Please fill in the following fields: ${missingFields.join(', ')}`);
+            return;
+        }
+
+        // Convert amount to number and validate
+        const amount = Number(formData.amount);
+        if (isNaN(amount)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)